packages/ui/src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,207 @@
+                        Newsroom
+                    </p>
+                </Link>
+                <div
+                    className={twMerge(
+                        'md:items-center justify-between gap-12 md:gap-4 hidden md:flex',
+                        Boolean(openMobileNav && isTablet) &&
+                            `flex flex-col w-screen absolute top-24 left-0 z-10 bg-white pt-6 border-b border-gray-200`,
+                        layout === 'centered' ? `md:w-2/3` : 'md:w-max',
+                    )}
+                >
+                    <div className="flex flex-col md:flex-row md:items-center gap-12 md:gap-4 px-6 md:px-0">
+                        {categories.length && (
+                            <CategoriesDropdown categories={categories} locale={locale} />
+                        )}
+                        {publicGalleriesCount > 0 && (
+                            <Link
+                                className={twMerge(
+                                    'label-large text-gray-600 hover:text-gray-800 shrink-0',
+                                    isTablet && `text-lg font-bold`,
+                                )}
+                                href="/media"
+                                locale={locale ?? false}
+                            >
+                                {/* TODO: Use translations */}
+                                Media
+                            </Link>
+                        )}
+                        {hasStandaloneAboutPage && (
+                            <Link
+                                className={twMerge(
+                                    'label-large text-gray-600 hover:text-gray-800 shrink-0',
+                                    isTablet && `text-lg font-bold`,
+                                )}
+                                href="/about"
+                                locale={locale ?? false}
+                            >
+                                {/* TODO: Use translations */}
+                                About
+                            </Link>
+                        )}
+                        {hasStandaloneContactsPage && (
+                            <Link
+                                className={twMerge(
+                                    'label-large text-gray-600 hover:text-gray-800 shrink-0',
+                                    isTablet && `text-lg font-bold`,
+                                )}
+                                href="/contacts"
+                                locale={locale}
+                            >
+                                {/* TODO: Use translations */}
+                                Contacts
+                            </Link>
+                        )}
+                    </div>
+
+                    <div className="flex flex-col md:flex-row md:items-center gap-12 md:gap-4">
+                        {Boolean(isSearchEnabled && onSearch) && (
```

Comment:
I wonder if we can combine these two props into one?
If you don't pass `onSearch` handler, the search would not be enabled anyway, so that makes `isSearchEnabled` prop kind of redundant 🤔